src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -101,8 +109,46 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
       }
   }
 
+  "CromwellApiService" should "return 404 for abort of unknown workflow" in {
+    Post(s"/workflow/${MockWorkflowManagerActor.unknownId}/abort") ~>
+      abortRoute ~>
+      check {
+        assertResult(StatusCodes.NotFound) {
+          status
+        }
+      }
+  }
+
+  it should "return 400 for abort of a malformed workflow id" in {
+    Post(s"/workflow/foobar/abort") ~>
+      abortRoute ~>
+      check {
+        assertResult(StatusCodes.BadRequest) {
+          status
+        }
+      }
+  }
+
+  it should "return 200 for abort of a known workflow id" in {
```

Comment:
Perhaps, depending on how our abort logic ends up being written.  Currently this is very simplistic and just validates that the supplied ID looks like a workflow ID and corresponds to a known "running" workflow.  There's currently no test for what the right behavior is when the ID corresponds to a known workflow in a terminal state.